complex.c

```diff
@@ -1850,16 +1853,35 @@ nucomp_to_r(VALUE self)
 
 /*
  * call-seq:
- *    cmp.rationalize([eps])  ->  rational
- *
- * Returns the value as a rational if possible (the imaginary part
- * should be exactly zero).
- *
- *    Complex(1.0/3, 0).rationalize  #=> (1/3)
- *    Complex(1, 0.0).rationalize    # RangeError
- *    Complex(1, 2).rationalize      # RangeError
- *
- * See to_r.
+ *   rationalize(epsilon = nil) -> rational
+ *
+ * Returns a Rational object whose value is exactly or approximately
+ * equivalent to that of <tt>self.real</tt>.
+ *
+ * With no argument +epsilon+ given, returns a \Rational object
+ * whose value is exactly equal to that of <tt>self.real</tt>:
```

Comment:
It calls the instance method `rationalize` on the real part.

```suggestion
 * whose value is exactly equal to that of <tt>self.real.rationalize</tt>:
```